docs/src/tutorials/connect-aws.md

```diff
@@ -0,0 +1,154 @@
+# Connect your device to AWS IoT
+
+The very first step to enable **thin-edge.io** is to connect your device to the cloud.
+* This is a 10 minutes operation to be done only once.
+* It establishes a permanent connection from your device to the cloud end-point.
+* This connection is secure (encrypted over TLS), and the two peers are identified by x509 certificates.
+* Sending data to the cloud will then be as simple as sending data locally.
+
+The focus is here on connecting the device to AWS IoT.
+For connecting to other clouds:
+
+- [Connecting to Cumulocity IoT](connect-c8y.md)
+- [Connecting to Azure IoT](connect-azure.md)
+
+Before you try to connect your device to AWS IoT, you need:
+
+- Navigate to AWS IoT Core console on AWS.
+- [Install `thin-edge.io` on your device](../howto-guides/002_installation.md).
+
+## Create the certificate
+
+The `tedge cert create` command creates a self-signed certificate which can be used for testing purpose.
+
+A single argument is required: an identifier for the device.
+This identifier will be used to uniquely identify your devices among others in your cloud tenant.
+This identifier will be also used as the Common Name (CN) of the certificate.
+Indeed, this certificate aims to authenticate that this device is the device with that identity.
+
+```shell
+$ sudo tedge cert create --device-id my-device
+```
+
+## Show certificate details
+
+You can then check the content of that certificate.
+
+```shell
+$ sudo tedge cert show
+Device certificate: /etc/tedge/device-certs/tedge-certificate.pem
+Subject: CN=my-device, O=Thin Edge, OU=Test Device
+Issuer: CN=my-device, O=Thin Edge, OU=Test Device
+Valid from: Tue, 09 Mar 2021 14:10:30 +0000
+Valid up to: Thu, 10 Mar 2022 14:10:30 +0000
+Thumbprint: 860218AD0A996004449521E2713C28F67B5EA580
+```
+
+You may notice that the issuer of this certificate is the device itself.
+This is a self-signed certificate.
+To use a certificate signed by your Certificate Authority,
+see the reference guide of [`tedge cert`](../references/tedge-cert.md).
+
+## Register the device on AWS IoT
+
+For a device to be trusted by AWS, one needs to add the self-signed certificate via the AWS IoT Console.
+
+- In the AWS IoT service sidebar, navigate to **Manage -> All devices -> Things**.
+- Click **Create things**
+- **Create single thing**
+- Enter the thing name which is the same as the `device-id` used in the step above
+- Select **Unnamed shadow (classic)**. For now, the shadow is used to verify successful connection and registration of
+  the device
+- If using self-signed certificate generated with `tedge`, **Use my certificate -> CA is not registered with AWS IoT**,
+  and upload the certificate in `/etc/tedge/device-certs`
```

Comment:
I feel it's good to automate this registration step as well. How hard is it to do that?